### PR TITLE
Task/bump plotly

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

Bump plotly libraries on client and backend to fix a plotting bug.
Note, other libraries were updated on the client side as well.

## Related Jira tickets: ##

None

## Testing Steps: ##
1. Check that bug is gone
2. Check other areas of applications as so many libraries were updated

## UI Photos:

Previously:
Screenshot 2023-01-17 at 4 44 13 PM

Update:
Screenshot 2023-01-20 at 4 19 58 PM

## Notes: ##

TODO: 
- [ ] looks like the demographic chart is taking up too much vertical space

### CODE REVIEW

File: protx-client/vite.config.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default defineConfig({
   },
   server: {
     host: 'cep.test',
+    port: 3000,
```
Comment (nathanfranklin) [inline protx-client/vite.config.ts:21]: @sophia-massie

**Review by taoteg — APPROVED** [review]
LGTM!

**Comment by nathanfranklin** [comment]
> looks like the demographic chart is taking up too much vertical space

 i still see this but think we should merge and address this somewhere else as not really related to the changes in this PR

**Comment by sophia-massie** [comment]
I'm not sure if @taoteg 's iframe PR will fix this, but the bottom of the modal is flush with the bottom of the screen and if I'm logged in to CMS, part of it is cut off and will not scroll. If this is a follow on issue to be resolved with the other PR, I can approve.

Screen Shot 2023-01-27 at 5 00 01 PM
Screen Shot 2023-01-27 at 5 00 42 PM
Screen Shot 2023-01-27 at 5 00 58 PM

**Comment by taoteg** [comment]
I agree. Maybe my iframe PR (when its updated) will resolve that.

**Review by sophia-massie — APPROVED** [review]
LGTM

**Pull request merged**